src/tool/subcommands/api_cmd.rs

```diff
@@ -1441,7 +1441,7 @@ fn eth_tests_with_tipset<DB: Blockstore>(store: &Arc<DB>, shared_tipset: &Tipset
         ),
         RpcTest::identity(EthGetBlockReceipts::request((block_hash.clone(),)).unwrap()),
         RpcTest::identity(
-            EthGetBlockTransactionCountByHash::request((block_hash.clone(),)).unwrap(),
+            EthGetBlockReceiptsLimited::request((block_hash.clone(), EthUint64(800))).unwrap(),
```

Comment:
Do not remove existing tests.

Reply:
Oh sorry, I was cloning the line instead it got removed